src/cmd_line/commands/jumps.ts

```diff
@@ -0,0 +1,55 @@
+import { window, QuickPickItem } from 'vscode';
+
+import * as node from '../node';
+import { VimState } from '../../state/vimState';
+import { globalState } from '../../state/globalState';
+import { Jump } from '../../jumps/jump';
+import { Range } from '../../common/motion/range';
+
+class JumpPickItem implements QuickPickItem {
+  jump: Jump;
+
+  label: string;
+  description?: string | undefined;
```

Comment:
These `| undefined`s are unnecessary because you've got the `?`s

Reply:
I have remove them.